autogpt_platform/backend/backend/copilot/briefing/narrative_test.py

```
@@ -0,0 +1,207 @@
+
+def make_content(run_items=None, **kwargs) -> BriefingContent:
+    defaults = dict(
+        generated_at=NOW,
+        timezone="UTC",
+        zero_expert_fallback=False,
+        run_items=run_items if run_items is not None else [make_run_item()],
+        decision_items=[],
+        decision_total=2,
+        completed_total=3,
+        failed_total=1,
+    )
```

Ruff (0.16.1)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)